tools/GlobalFilters.xml

```diff
@@ -162,6 +162,7 @@
         <string>ServiceManagement\\Common\\Commands.ScenarioTest\\Resources\\RdfeTestDummy.publishsettings</string>
         <string>ServiceManagement\\Common\\Commands.ScenarioTest\\Resources\\ChefExtension\\client.rb</string>
         <string>ServiceManagement\\Common\\Commands.ScenarioTest\\Resources\\ChefExtension\\tstorgnztn-validator.pem</string>
+        <string>ServiceManagement\\Common\\Commands.ScenarioTest\\Resources\\ChefExtension\\encrypted_data_bag_secret</string>
```

Comment:
Please ensure that this new file does not represent a credentials used anywhere in azure - it would be a security breach to include a real credential.